Mage.Sets/src/mage/sets/MysteryBooster.java

```diff
@@ -44,3487 +41,1918 @@ public static MysteryBooster getInstance() {
-                "Revel in Riches",
-                "Rune-Scarred Demon",
-                "Savage Knuckleblade",
-                "Selvala, Heart of the Wilds",
-                "Serendib Efreet",
-                "Sewer Nemesis",
-                "Shamanic Revelation",
-                "Sliver Hivelord",
-                "Solemn Simulacrum",
-                "Spawning Grounds",
-                "Star of Extinction",
-                "Steamflogger Boss",
-                "Stunt Double",
-                "Sudden Demise",
-                "Supreme Verdict",
-                "Sword of the Animist",
-                "Talrand, Sky Summoner",
-                "Taurean Mauler",
-                "Teferi, Temporal Archmage",
-                "Teferi's Protection",
-                "Temporal Mastery",
-                "Tempt with Discovery",
-                "Thalia's Lancers",
-                "The Gitrog Monster",
-                "The Mirari Conjecture",
-                "Tireless Tracker",
-                "Torment of Hailfire",
-                "Trading Post",
-                "Two-Headed Giant",
-                "Urza's Rage",
-                "Vigor",
-                "Wheel of Fate",
-                "Whelming Wave",
-                "Whir of Invention",
-                "Yuriko, the Tiger's Shadow"
-        ));
-        this.populateBoosterSpecialSlot();
-    }
-
-    /**
-     * Populate the given booster slot.
-     *
-     * @param slotNumber booster slot number. 1-indexed, valid range is 1-14, as 15 is the special slot
-     * @param cardNames  List of English card names found on the given slot
-     */
-    private void populateSlot(int slotNumber, List<String> cardNames) {
-        final List<CardInfo> cardInfoList = this.possibleCardsPerBoosterSlot.get(slotNumber);
-        for (String name : cardNames) {
-            final CardInfo cardWithGivenName = CardRepository.instance.findCardWithPreferredSetAndNumber(name, this.code, null);
-            cardInfoList.add(cardWithGivenName);
-        }
-    }
-
-    /**
-     * Populate the special slot. Defined as protected to allow overwriting this if the playtest cards
-     * from the convention edition are ever implemented.
-     */
-    protected void populateBoosterSpecialSlot() {
-        CardCriteria criteria = new CardCriteria();
-        criteria.setCodes("FMB1");
-        this.possibleCardsPerBoosterSlot.get(15).addAll(CardRepository.instance.findCards(criteria));
+        // This set is implemented for the purpose of booster generation only.
+        // Rather than include cards (scryfall moved them to PLST with thousands of other cards) with duplicate printings,
+        // the collation just draws from the original printings in other sets.
+        // However, this set would be excluded from view if it contained no cards, so these two cards are here as a workaround.
+        cards.add(new SetCardInfo("Goblin Trenches", 203, Rarity.RARE, mage.cards.g.GoblinTrenches.class));
```

Comment:
> I couldn't find what part of the code excludes the set from display if it doesn't contain any cards, though, so I included a couple as a workaround.

Can you explain what's wrong with empty set? It's ok to show it in set's list, card viewer, downloader, etc (from UX logic). No need to exclude.

If you see some errors or weird drawing then related code must be improved (e.g. showing/mark set as virtual/remix) - must create new issue about it with all founded problems.

Reply:
With the empty set:
- Still shows in card viewer 0 of 0 with no issues 
- Does not appear in dropdown for selecting booster packs to draft, which is exactly where it's needed
- Also doesn't appear for choosing sets for random booster draft